frontend/src/api/Socket.ts

```diff
@@ -84,16 +84,16 @@ export const send = (sendUrl: string, headers?: {}, body?: string):
 
 /**
  * Disconnect the user by sending a message via socket.
- * @returns void, or error if socket is not connected.
+ * @returns void, or do nothing if socket is not connected.
 */
 export const disconnect = ():
-  Promise<void> => new Promise<void>((resolve, reject) => {
+  Promise<void> => new Promise<void>((resolve) => {
     if (stompClient && stompClient.connected) {
       // Stomp client is already set from last connection.
       stompClient.disconnect(() => {
         resolve();
       });
     } else {
-      reject(errorHandler('The socket is not connected.'));
+      resolve();
     }
   });
```

Comment:
Sure - do you mean like to connect them if they aren't connected already? In which case that might also make the connect function redundant 

As for this place, the main reason I did this change was so that disconnect would basically ensure you're not connected to the socket (regardless of whether you were connect or not in the first place)

Reply:
Oh I meant that you replaced the `disconnect` function's `reject(errorHandler('The socket is not connected.'));` with `resolve();`, whereas `subscribe` and `send` still have the `reject` lines; so I'm thinking either change all to `resolve();` or leave everything as-is.

Reply:
The reason I went with this was cause calling disconnect when already disconnected doesn't change anything (and thus I didn't think it warranted an error message) whereas calling subscribe/send when not connected would definitely cause an error to come up - however, the disconnect reject error that originally prompted this change I think was fixed elsewhere so I'd be happy to change them all back to reject if we wanted to prioritize consistency

Reply:
Yeah, I think that'd make sense to change it back, if that's alright 👍